<launch>

<param name="/calibration/config_file_rgb_l" value="/datasets/rgbd_tof_fusion_output/src/camOdoCalib/config/vicore_f35_l.yaml" type="string"/>
<param name="/calibration/config_file_rgb_r" value="/datasets/rgbd_tof_fusion_output/src/camOdoCalib/config/vicore_f35_r.yaml" type="string"/>  

<!-- initial extrinsic param Tdepth_rgb, pose from rgb cam to depth cam -->
<!-- <param name="/Tdepth_rgb/tx" value="-0.0473584" type="double"/>
<param name="/Tdepth_rgb/ty" value="0.0544932" type="double"/>
<param name="/Tdepth_rgb/tz" value="-0.0187574" type="double"/>
<param name="/Tdepth_rgb/qx" value="0.0141265" type="double"/>
<param name="/Tdepth_rgb/qy" value="0.02082" type="double"/>
<param name="/Tdepth_rgb/qz" value="0.002037" type="double"/>
<param name="/Tdepth_rgb/qw" value="0.999681" type="double"/> -->

<node pkg="cam_calibration" type="stereo_april_check_node" name="stereo_april_check" output="screen">
    <!-- <remap from="cam0/image_raw" to="/cam0/image_raw"/>
    <remap from="cam1/image_raw" to="/cam1/image_raw"/> 
    <remap from="camera/depth/image_raw" to="/camera/depth/image_raw"/> -->
</node>

<node pkg="rviz" type="rviz" name="rviz" output="screen" args="-d $(find cam_calibration)/rviz/vins_rgbd.rviz"/>
</launch>
